src/assets/prosemirror/menu.css

```diff
@@ -0,0 +1,147 @@
+.ProseMirror-textblock-dropdown {
```

Comment:
This turned into a big scoping problem and needing to add deep selectors on everything. I'm gonna leave it as global css.